languageservice/src/code-actions/types.ts

```diff
@@ -0,0 +1,21 @@
+import {CodeAction, Diagnostic} from "vscode-languageserver-types";
+
+export interface CodeActionContext {
+  uri: string;
+  // TODO: add things like workflow template, parsed content, etc.
```

Comment:
We may want to be careful about adding more things here, since removing or changing the structure would be a compat break

Reply:
Wondering if we should remove this TODO ?